example.dev.env

```diff
@@ -14,6 +14,8 @@ bCMS__DB_DATABASE=adamrms
 bCMS__DB_USERNAME=root
 bCMS__DB_PASSWORD=root
 
+bCMS__ASSETS_URL=https://assets.adam-rms.com/file/AdamRMS-assets
```

Comment:
This also needs adding to the deployment https://github.com/bstudios/adam-rms-deployment/commit/de5717d962215d635bc65bcfdf9450feb9682655